DeviceTests/DeviceTests.Shared/Email_Tests.cs

```diff
@@ -1,12 +1,32 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
+﻿using System.Threading.Tasks;
 using Xamarin.Essentials;
 using Xunit;
 
 namespace DeviceTests
 {
+    // TEST NOTES:
+    //   - a human needs to close the email composer window
     public class Email_Tests
     {
+        [Fact]
+        [Trait(Traits.InteractionType, Traits.InteractionTypes.Human)]
+        public async Task Compose_Shows_New_Window()
+        {
+            await Email.ComposeAsync();
+        }
```

Comment:
The new attribute allows test to be written that not only require input, but also leave the app entirely.